Makefile

```diff
@@ -94,6 +100,12 @@ vulncheck: ## Verify code vulnerabilities
 codegen: ## Auto generate files based on Azure API responses
 	./hack/codegen.sh
 
+snapshot: ## Builds and publishes snapshot release
```

Comment:
In most cases this is because the updates the chart version & values, so the next run will detect uncommited changes and refuse to proceed. Opting to add a message that clarifies what happened, rather than any kind automatic cleanup, which should help interactive scenarios. In GH action (which should drive this in most cases) this should not be a problem.